index.html

```diff
@@ -1,5 +1,4 @@
----
```

Comment:
You have removed these dashes which are important for jekyll add them and things will work out